install.md new

Avani8 opened the pull request:
changing install.md

flihp commented:
This work started in #108 and is now continuing here.

flihp commented:
Hey @Avani8. I've taken your patch from this PR and done some significant re-writing. Specifically I've removed the numbered sections you added, made the line length consistent and made a number of stylistic changes. I've also added a meaningful commit message (which makes my life as a maintainer a lot easier).

I can't push my work directly to your PR so I've created a new one in #118. Since I've left your name in the 'Author' field I figure you should review my changes to be sure they don't change the intent of your work (which was hard to divine since you didn't provide a commit message :wink:).

I'm closing this issue but please do review my derivative work in #118 so we can close this out. Thanks!

Avani8 commented:
Hi Philip,


thank you so much for giving me this opportunity and sorry for not writing
commit message as I am totally new for this task. really it was your great
help in pulling this thing off. I've reviewed #118 and it looks fine to me.


Regards,

Avani Dave
4084971448

On Mon, Jul 31, 2017 at 2:11 PM, Philip Tricca <notifications@github.com>
wrote:

> Hey @Avani8 <https://github.com/avani8>. I've taken your patch from this
> PR and done some significant re-writing. Specifically I've removed the
> numbered sections you added, made the line length consistent and made a
> number of stylistic changes. I've also added a meaningful commit message
> (which makes my life as a maintainer a lot easier).
>
> I can't push my work directly to your PR so I've created a new one in #118
> <https://github.com/01org/tpm2-abrmd/pull/118>. Since I've left your name
> in the 'Author' field I figure you should review my changes to be sure they
> don't change the intent of your work (which was hard to divine since you
> didn't provide a commit message 😉).
>
> I'm closing this issue but please do review my derivative work in #118
> <https://github.com/01org/tpm2-abrmd/pull/118> so we can close this out.
> Thanks!
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/01org/tpm2-abrmd/pull/116#issuecomment-319150506>, or mute
> the thread
> <https://github.com/notifications/unsubscribe-auth/ALe9DD39giAldcMeSlArcjzuLZVNPZu_ks5sThjigaJpZM4OnDmc>
> .
>

flihp commented:
No worries @Avani8. It's a learning process. Your work here has already provided a solution for the question just asked in #115. Thanks!

Avani8 commented:
Great !!! Thank you

On Mon, Jul 31, 2017 at 3:35 PM Philip Tricca <notifications@github.com>
wrote:

> No worries @Avani8 <https://github.com/avani8>. It's a learning process.
> Your work here has already provided a solution for the question just asked
> in #115 <https://github.com/01org/tpm2-abrmd/issues/115>. Thanks!
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/01org/tpm2-abrmd/pull/116#issuecomment-319172948>, or mute
> the thread
> <https://github.com/notifications/unsubscribe-auth/ALe9DHIdOno9YGX4L0E6SNXRO1zM-0LNks5sTiyAgaJpZM4OnDmc>
> .
>
-- 
Regards,

Avani Dave
4084971448
